bleachbit/Cleaner.py

```diff
@@ -134,6 +134,19 @@ def get_option_descriptions(self):
         if self.options:
             for key in sorted(self.options.keys()):
                 yield (self.options[key][0], self.options[key][1])
+                
+    def get_long_option_description(self, key):
+        """Yield the name and a potentially more verbose description for a single option"""
```

Comment:
Under the System category there are no actions listed. Either they should be listed, or the introductory phrase "this cleanup option" should be omitted.

Likewise for the Firefox cleaner. (I have recently done prep work to move it CleanerML, which would fix this, but that is not quite ready.)